README.md

```diff
@@ -1,59 +1,245 @@
+
+Ethereum 2.0 is the name that has been given to a new blockchain being
+developed by the Ethereum Foundation and the Ethereum community. Ethereum 2.0
+consists of 1,025 proof-of-stake blockchains; the "beacon chain" and 1,024
+"shard chains".
+
+## Beacon Chain
+
+The Beacon Chain is a little different to more common blockchains like Bitcoin
+and present-Ethereum in that it doesn't process "transactions", per say.
+Instead, it maintains a set of bonded (staked) validators and co-ordinates them
+to provide services to a static set of "sub-blockchains" (shards). These shards
+then process the normal "5 ETH from A to B" transactions in _parallel_ whilst
+deferring consensus to the Beacon Chain.
+
+Here are some of the major services that the beacon chain provides to its
+shards:
+
+- A source of entropy, likely using a [RANDAO + VDF
+  scheme](https://ethresear.ch/t/minimal-vdf-randomness-beacon/3566).
+- Valdidator management, including:
+    - Inducting and ejecting validators.
+    - Delegating randomly-shuffled subsets of validators to validate shards.
+    - Penalising and rewarding validators.
+- Proof-of-stake consensus for shard chain blocks.
+
+## Shard Chains
+
+Shards can be thought of like CPU cores, they're a lane where transactions can
+execute in series (one-after-another). Presently, Ethereum is single-core and
+can only _fully_ process one transaction at a time. Sharding allows multiple
+transactions to happen in parallel, greatly increasing the per-second
+transaction capacity of Ethereum.
+
+Each shard is proof-of-stake and shares its validators (stakers) with the other
+shards as the beacon chain rotates validators pseudo-randomly across shards.
+Shards will likely be the basis of very interesting layer-2 transaction
+processing schemes, however we won't get into that here.
+
+## The Proof-of-Work Chain
+
+The proof-of-work chain will hold a contract to allow addresses to deposit 32
+ETH, a BLS public key and some [other
+parameters](https://github.com/ethereum/eth2.0-specs/blob/master/specs/casper_sharding_v2.1.md#pow-chain-changes)
+to allow them to become Beacon Chain validators. Each Beacon Chain will
+reference a PoW block hash allowing PoW clients to use the Beacon Chain as a
+source of [Casper FFG finality](https://arxiv.org/abs/1710.09437), if they
+desire.
+
+## Ethereum 2.0 Progress
+
+Ethereum 2.0 is not fully specified and there's no working implementation. Some
+teams have demos available, however these demos indicate _progress_, not
+completion. We look forward to providing user functionality once we are ready
+to provide a minimum-viable user experience.
+
+The work-in-progress specification lives
+[here](https://github.com/ethereum/eth2.0-specs/blob/master/specs/casper_sharding_v2.1.md)
+in the [ethereum/eth2.0-specs](https://github.com/ethereum/eth2.0-specs)
+repository. It is still in a draft phase, however there are several teams
+already implementing it. This spec is being developed by Ethereum Foundation
```

Comment:
The specification is still in draft phase, however, there are several teams
 already implementing it.

- initial "It" was ambiguous

Reply:
Had to reword a bunch of this, so I fixed this too :)